Four Brains/Model/DrumSounds.swift

```diff
@@ -177,29 +204,29 @@ class DrumSounds {
     }
     func playDrumSounds() {
         
-        do {
-            try AKSettings.setSession(category: .playAndRecord, with:  AVAudioSession.CategoryOptions.defaultToSpeaker)
-                
-            let session = AVAudioSession.sharedInstance()
-            try session.setCategory(AVAudioSession.Category.playAndRecord)
-
-            if !AKSettings.headPhonesPlugged {
-                try session.overrideOutputAudioPort(AVAudioSession.PortOverride.speaker)
-         }
-        }catch {
-            print("error in settings.setSession")
-        }
+//        do {
```

Comment:
How come this is commented out?

Reply:
Maybe leave a TODO: ... get back to this later, not working right now...

Reply:
I went ahead and left the TODO. Thanks!